### DISCUSSION TITLE
Ignore .gradle directory

### DISCUSSION THREAD

**Discussion by JajaComp** [LemonAppDev/konsist#512] [Report a Problem] [answered]
### Tell us what version of Konsist you use.

0.11

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

In gitlab cicd when i use test from snippet i got issue:

<failure message="com.lemonappdev.konsist.core.exception.KoCheckFailedException: Assert 'no wildcard imports allowed' has failed. Invalid declarations (936):&#10;/builds/user/project/.gradle/caches/8.3/kotlin-dsl/accessors/07d0bccad7951fbc49a5af18e38b7927/sources/org/gradle/kotlin/dsl/Accessors14w8c0izd80vth0z04l852wsa.kt:57:1 (org.gradle.kotlin.dsl ImportDeclaration)&#10;/builds/user/project/.gradle/caches/8.3/kotlin-dsl/accessors/07d0bccad7951fbc49a5af18e38b7927/sources/org/gradle/kotlin/dsl/Accessors14w8c0izd80vth0z04l852wsa.kt:58:1 (org.gradle.kotlin.dsl.accessors.runtime ImportDeclaration)&#10;/builds/user/project/.gradle/caches/8.3/kotlin-dsl/accessors/07d0bccad7951fbc49a5af18e38b7927/sources/org/gradle/kotlin/dsl/Accessors15c80wxji7qdhvcsteytc58m6.kt:57:1
etc...

Snipped:
```
    @Test
    fun `no wildcard imports allowed`() {
        Konsist
            .scopeFromProject()
            .imports
            .assertNot { it.isWildcard }
    }
```

### Actual behaviour

Test check .gradle directory

### Expected behaviour

Ignore .gradle directory

**Comment by igorwojda** [comment, marked as answer]
Hey, thx for the repost. This was a bug on our side. it was just fixed https://github.com/LemonAppDev/konsist/pull/523 and will go with the next release (this week).
